utils.py

```diff
@@ -321,7 +321,7 @@ def handle_author_import(row, article):
         row.get('Author surname'),
         row.get('Author institution'),
         row.get(''),
-        row.get('Author email'),
+        row.get('Author email').strip(),
```

Comment:
Since `row.get('Author email')` could return `None`, we can't always call `.strip` on the return value. Instead, we need to check if there is a value for this row first.